views/layouts/HeroLayout/index.jsx

```diff
@@ -11,7 +11,10 @@ const HeroLayout = () => {
     <Box className="HeroLayout_container">
       <Box className="HeroLayout_main">
         <HeroCard />
-        <Outlet />
+
+        <Box sx={{ width: '458px', boxShadow: 5, px: '64px', py: '50px' }}>
```

Comment:
There are different widths for the different forms, I think the width should be specified by the container of the form that belongs to the outlet.

Reply:
the layout sets up the page that the form renders into. In the case of the HeroLayout, it is only the right side box that is customized by the pages. This change makes it so the box size is consistent and we no longer have to specify dimensions in the pages, they just fill up this space.

Reply:
If we define the width here and then we require a different width for another form/page it could be thing

Reply:
all hero layouts have this layout. I don't see why we'd want to specify it on every other page and run the risk of them being different which would make the UI inconsistent.

Reply:
the other screen has a different width is why i'm mentioning it

Reply:
this seems to be a mistake in the designs. Erica confirmed they should have consistent widths https://gnardog.slack.com/archives/C05DT214NHY/p1699556749283799?thread_ts=1699556549.223369&cid=C05DT214NHY